internal/cmd/beta/security-group/list/list.go

```diff
@@ -0,0 +1,170 @@
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			projectLabel, err := projectname.GetProjectName(ctx, p, cmd)
+			if err != nil {
+				p.Debug(print.ErrorLevel, "get project name: %v", err)
+				projectLabel = model.ProjectId
+			}
+
+			// Call API
+			request := buildRequest(ctx, model, apiClient)
+
+			response, err := request.Execute()
+			if err != nil {
+				return fmt.Errorf("list security group: %w", err)
+			}
+			p.Info("Listed security group %v for %q\n", model.LabelSelector, model.ProjectId)
+
+			if items := response.GetItems(); items == nil || len(*items) == 0 {
+				p.Info("no security groups found for %q", projectLabel)
+			} else {
+				if err := outputResult(p, model.OutputFormat, *items); err != nil {
+					return fmt.Errorf("output security groups: %w", err)
+				}
+			}
+
+			return nil
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().String(labelSelectorFlag, "", "Filter by label")
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		LabelSelector:   flags.FlagToStringPointer(p, cmd, labelSelectorFlag),
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *iaas.APIClient) iaas.ApiListSecurityGroupsRequest {
+	request := apiClient.ListSecurityGroups(ctx, model.ProjectId)
+	if model.LabelSelector != nil {
+		request = request.LabelSelector(*model.LabelSelector)
+	}
+
+	return request
+}
+func outputResult(p *print.Printer, outputFormat string, items []iaas.SecurityGroup) error {
+	switch outputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(items, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal PostgreSQL Flex instance list: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	case print.YAMLOutputFormat:
+		details, err := yaml.MarshalWithOptions(items, yaml.IndentSequence(true))
+		if err != nil {
+			return fmt.Errorf("marshal PostgreSQL Flex instance list: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		table := tables.NewTable()
+		table.SetHeader("ID", "NAME", labelSelectorFlag, "STATEFUL")
+		for _, item := range items {
+			table.AddRow(ptrString(item.Id), ptrString(item.Name), concatLabels(*item.Labels), ptrString(item.Stateful))
+		}
```

Comment:
We usually keep the columns in the `list` table to the main ones used to list a resource. So I would not show the `labels` here, and leave it for the `describe`